freedomofpress/securedrop.org
# CAPTCHA Fixes

How to test:

1. Visit the contact form (or create your own FormPage to visit in the wagtail admin)
2. Observe in the browser console the _absence_ of an exception like

   ```
   Content Security Policy: The page’s settings blocked the loading of a resource at https://www.google.com/recaptcha/api.js?hl=en (“script-src”).
   ```
3. Fill out the form and ensure that it submits successfully
